public/data/RohithJN.json

```
@@ -0,0 +1,39 @@
+{
+    "name": "Rohith JN",
+    "type": "personal",
+    "bio": "Junior developer // React.js, Python, APIs",
+    "avatar": "https://github.com/Rohith-JN.png",
+    "links": [
+      {
+        "name": "Follow me on GitHub",
+        "url": "https://github.com/Rohith-JN",
+        "icon": "github"
+      },
+      {
+        "name": "Follow me on Twitter",
+        "url": "https://twitter.com/RohithNambiar4",
+        "icon": "twitter"
+      },
+      {
+        "name": "Connect with me on Linkedin",
```

I recommend capitalizing `i` here in `LinkedIn`. What do you think!? 
```suggestion
        "name": "Connect with me on LinkedIn",
```